test/DePayRouterV1CurveFiSwap01.spec.ts

```diff
@@ -0,0 +1,63 @@
+import chai, { expect } from 'chai'
+
+import { solidity, loadFixture } from 'ethereum-waffle'
+import { BigNumber } from 'ethers'
+
+import {
+  cureFiFixture,
+  cureFiSwapFixture,
+  curveFiSystemFixture,
+  unapprovedCureFiFixture
+} from './fixtures/curvefi-swap'
+
+import { route as routerFunc } from './functions'
+
+import { now, ETH } from './utils'
+
+chai.use(solidity)
+
+describe('CurveFiSwap01', () => {
```

Comment:
Let's call this `DePayRouterV1 + DePayRouterV1CurveFiSwap01` to be symmetrical with the naming of the other tests: e.g. https://github.com/DePayFi/depay-ethereum-router/blob/master/test/DePayRouterV1Uniswap01.spec.ts#L21